static/js/main.js

```diff
@@ -464,9 +468,16 @@ function initializeHotkeys() {
     });
 }
 
+function removeKeydownSideEffects() {
```

Comment:
I don't like this function name, it is both too specific and ambiguous.

Too specific: "removeKeydown".
We may choose another key event type (e.g. keyUp).

Ambiguous: "SideEffects".
["Side effect"](https://en.wikipedia.org/wiki/Side_effect_(computer_science)) has a very specific meaning in CS,
and what is happening here is not a CS side effect or even technically a side effect in the looser sense (it is unintentional, but not a side effect).

Mirroring your initializeHotkeys() function, maybe something like "removeHotkeysOnText()".